### DISCUSSION TITLE
Main Website- Division of pages

### DISCUSSION THREAD

**Discussion by Aaditya18SP** [rushiii3/KETS#21] [Ideas]
# This is the discussion of how are we going to divide the pages and assign them accordingly

I feel we should divide it tab wise because each tab consists of pages that should look similar

### eg: 1. 
In ***About us*** tab, the pages under ***Governance*** tab ie ***Legacy, College Mgmt, College Administration*** will all look similar mostly.
***NIRF*** tab is the odd man out here.And the rest are PDFs which should be a easy for anyone to do. 

### eg 2: 
In ***Academics*** tab, the course catalog pages  (***Junior College, UG courses, PG courses,Phd***) are all similar and only the data that will be fetched will change. 
The ***Departments, GD Kelkar Skill*** page will be different from one another.

**Comment by Aaditya18SP** [comment]
# Division of Tabs- Idea

1. 2 people will work on one tab
2. the team will be consist one from @Aaditya18SP,  @rushiii3 and one from @NitinSingh0, @AniketSingh52 .
3. The team that finishes their tab before another will choose the next tab the other team will choose.
4. After a team completes their tab, the teams will be switched.  

--------

# Why this way?

It ensures consistency in the website. Instead of having 4 different looking tabs will at worst have 2  different looking versions. Also there will be discussion among the team members instead of everyone working solo, so we will be updated as to how much we are progressing

**Reply by NitinSingh0** [reply]
Alright, assign one tab to each team.

**Comment by NitinSingh0** [comment]
Could you please upload the new nav bar file?
